md-dao/src/main/java/com/wipro/ats/bdre/md/dao/JobDAO.java

```diff
@@ -41,15 +41,24 @@
     private static final Logger LOGGER = Logger.getLogger(JobDAO.class);
     @Autowired
     SessionFactory sessionFactory;
-
-
-    public List<InitJobRowInfo> initJob(Integer processId, Integer maxBatch) throws Exception {
+    private static final String PROCESS="process";
+    private static final String ENQUEUINGPROCESSID="enqueuingProcessId";
+    private static final String DELETEFLAG="deleteFlag";
+    private static final String BATCHCUTPATTERN="batchCutPattern";
+    private static final String PROCESSID="processId";
+    private static final String INVALIDPARENTPROCESS="Invalid parent-process. pid=";
+    private static final String ENDTS="endTs";
+    private static final String EXECSTATUS="execStatus";
+    private static final String BATCHBYTARGETBATCHID="batchByTargetBatchId";
+    private static final String BATCHBYSOURCEBATCHID="batchBySourceBatchId";
+    private static final String NOTUNDEREXECUTIONPROCESS="The process is not under execution, pid=";
+    public List<InitJobRowInfo> initJob(Integer processId, Integer maxBatch) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "initJob" is 64 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)